src/entity/index.ts

```diff
@@ -32,6 +34,17 @@ export class IasqlDatabase {
     default: false,
   })
   directConnect: boolean;
+
+  @Column({
+    type: 'int',
+  })
+  megabytes: number;
+
+  @CreateDateColumn()
+  createdAt: Date;
+
+  @UpdateDateColumn()
+  updatedAt: Date;
```

Comment:
Does this updates on every change automatically?

Reply:
yes

Reply:
Well, it does in the ORM. It's clear in the migration that Postgres itself doesn't have this functionality, unfortunately.